interface/client/appStart.js

```diff
@@ -24,7 +24,7 @@ mistInit = function(){
 
             Tabs.insert({
                 _id: 'browser',
-                url: 'https://ethereum.org',
+                url: 'http://dapps.ethercasts.com/',
```

Comment:
@alexvandesande who takes care of dapps.ethercasts? Should we contribute with them to have a nicer (and less colorful) layout? 
Will this be officially the Mist first page?